### ISSUE TITLE
Seed of Corruption issue

### ISSUE THREAD

**Issue by moooreno** [gamefreedomgit/Frostmourne#1291]
Today i met spell isse on seed of corruption. Casting it into pack of few mobs but when they explode hitting just 1 target instead all of them. Check my MSBT dmg after explode (should be x5 if 5 targets for example)

Screenshot ---> https://ibb.co/DtW2zLX

**Comment by disruption01** [comment]
I play warlock, never had an issue with seed of corruption, working correctly. In your screenshot I don't see anything wrong lol

**Comment by scrublama** [comment]
What I've noticed from raids is my seeds sometimes not dealing dmg when there are other locks seeding.
So I went to PTR to try it.

When multiple seeds are cast on a target they will both explode whenever the first one meets the threshold.
When this happens only 1 seed will give explosion damage to x player, it seems to be random who gets the damage as in; It doesn't seem to matter who applied it first, whos the stronger or who triggered the explosion.
https://vimeo.com/982005131?share=copy
